uint8 event
uint8 RELEASED = 0
uint8 PRESSED = 1
